user_deletion/managers.py

```diff
@@ -0,0 +1,22 @@
+from dateutil.relativedelta import relativedelta
+from django.apps import apps
+from django.utils import timezone
+
+
+user_deletion_config = apps.get_app_config('user_deletion')
+
+
+class UserManagerMixin:
+    def users_to_notify(self):
+        """Finds all users who have been inactive and not yet notified."""
+        last_login = timezone.now() - relativedelta(
+            months=user_deletion_config.MONTH_NOTIFICATION,
+        )
```

Comment:
`boundary`? `notification_boundary`? `inactive_boundary`? It's not a very easy one to name I agree.